lisa/messages.py

```diff
@@ -125,6 +125,22 @@ class PerfMessage(MessageBase):
     role: str = ""
     test_result_id: str = ""
 
+    # Sysbench common params
```

Comment:
The PerfMessage is a base class. Please either move them into existing perf message, or new perf messages.

Reply:
The perf messages should be metrics oriented, instead of tool oriented. It's easy to create dashboard and explain by metrics concepts. It's the same to sysbench common fields. It looks many common fields are not help a lot as a "common" field, the tested metrics are the key information of a perf message. We need to abstract the values to perf metrics.

For example, it's easy to understand x_latency_ms in network tests, but hard to understand in CPU tests, and I'm not sure what does it mean in the mem tests. The "events" number is highly depends on the test scenarios and settings.

Reply:
If you look at the sysbench output it will always give latency metrics like min/max/avg. That was the reason to put it under common sysbench metrics